script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1.txt

```diff
@@ -57,10 +57,10 @@ The past seven years have been truly peaceful. As long as I didn't cause trouble
 <Page17>
 "After seven years, I'm as healthy as a man can be.
 Even though it looks like this, my body is rather sturdy, you know."
-"Yes, you are right.
-The Tohnos are very different from other families, but you have never seemed to be afraid of them. Ever since you were a child, you have been so fearless that it amazes me."
+"Yes, you're right.
+The Tohnos are very different from other families, but you never seemed to be afraid of them. Ever since you were a child, you've been so fearless that it amazes me."
 <Page18>
-At her words mingled with a bitter smile, I smiled back.
+At her words that mingle with a bitter smile, I smiled back.
```

Comment:
I think I prefer the previous wording here

Reply:
I was changing it to present tense, might not be necessary here though. Mingling, maybe?